packages/daemon/src/generator/keras.ts

```diff
@@ -0,0 +1,74 @@
+import { exec } from 'child_process';
+import * as path from 'path';
+import * as fs from 'fs-extra';
+import { GenerateOptions } from '../service/pipeline';
+
+const boa = require('@pipcook/boa');
+
+export async function keras2Wasm(dist: string, projPackage: any, opts: GenerateOptions, inputLayer?: string) {
+  const relay = boa.import('tvm.relay');
+  const emcc = boa.import('tvm.contrib.emcc');
+  const keras = boa.import('tensorflow.keras');
+  const { dict, open, len } = boa.builtins();
+  exec(`wget http://ai-sample.oss-cn-hangzhou.aliyuncs.com/tvmjs/dist/tvmjs.bundle.js`, {cwd: dist});
+  exec(`wget http://ai-sample.oss-cn-hangzhou.aliyuncs.com/tvmjs/preload.js`, {cwd: dist});
```

Comment:
Why not using Node.js to download them?

Reply:
And we don't need to download them every time actually, and the above files should be versioned, too.

Reply:
Caching involves complex logic on version control and those files are relatively stable and small. For now, I think it is safe to download every time. We could fix it in another PR.

Reply:
There is no complexity to implement that, rename filenames with a specific version, and download them with a cache in memory.